README.md

```diff
@@ -1,75 +1,70 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Getting Started with Create React App
```

Comment:
- [ ] Kindly format your README using [the Microverse template](https://github.com/microverseinc/readme-template)